README.md

```diff
@@ -23,23 +23,34 @@ Connecting those modules with each other lets you create immensly useful agents.
 
 ## Installation
 
-Beehive requires Go 1.8 or higher. Make sure you have a working Go environment. See the [install instructions](http://golang.org/doc/install.html).
+Beehive requires Go 1.8 or higher. Make sure you have a working Go environment.
+See the [installation instructions of Go](http://golang.org/doc/install.html).
 
 ### From source
 
-The recommended way is to fetch the sources and run make. This requires [dep](https://github.com/golang/dep)
-to be installed on your system.
+The recommended way is to fetch the sources and run make.
+This requires [dep](https://github.com/golang/dep) to be installed on your system.
 
+If the `$GOPATH` environment variable is empty (`echo $GOPATH`), set it with:
+    
+    # Set the $GOPATH environment variable for the active shell
+    export GOPATH="/your/go/path"
```

Comment:
@LorenzoAncora given that `GOPATH` is now optional in go 1.8+ and set to `~/go` by default, I'm wondering if we should just tell people to use `~/go` to simplify the instructions and avoid the export.

Reply:
We should instead just upgrade the deps and docs  to be compatible with Go11Modules.  It's far easier and  just works. One does not need  to care about  any `$GOPATH`.